README.md

```diff
@@ -89,7 +94,8 @@ Change Log
 ==========
 
 Version 2.7.2 : June 6, 2017
-- Fixed assembly versioning.
+- Fixed assembly versioning
```

Comment:
Maybe a bit more explicit: Fixed issue where assembly version wasn't set in assembly